src/main/scala/tech/cryptonomic/conseil/tezos/TezosPlatformDiscoveryOperations.scala

```diff
@@ -107,13 +107,11 @@ object TezosPlatformDiscoveryOperations {
   }
 
   /** Checks if columns exist for the given table */
-  def areFieldsValid(tableName: String, queryFields: List[String], predicateFields: List[String]): Boolean = {
-    val fields = (queryFields ++ predicateFields).toSet
-
+  def areFieldsValid(tableName: String, fields: List[String]): Boolean = {
```

Comment:
Would it be useful to require a `fields: Set[String]` in the first place, as a parameter?

Reply:
It could, but if I remember correctly this is the only place I use `Set` features

Reply:
what I meant is that putting that as a "type requirement" in the method signature will make for a  more specific and clearly defined function semantic

Reply:
I'm not convinced but I've done it 😄